selftests/functional/test_loader.py

```diff
@@ -154,7 +154,7 @@ def _test(self, name, content, exp_str, mode=MODE_0664, count=1):
                                              'avocado_loader_test',
                                              mode=mode)
         test_script.save()
-        cmd_line = ('%s list -V %s' % (AVOCADO, test_script.path))
+        cmd_line = ('%s -V list %s' % (AVOCADO, test_script.path))
```

Comment:
I know you have not completed the refactoring yet, but just an FYI that this change is also needed here: `optional_plugins/loader_yaml/tests/test_yaml_loader.py` as Travis is failing with `unrecognized arguments: -V`.

Reply:
Fixed, will mark this as resolved.